.github/workflows/deploy-testing.yml

```diff
@@ -51,15 +48,9 @@ jobs:
           BUILD_NUMBER: ${{github.run_number}}
           API_URL: ${{ secrets.DGC_BASE_URL }}
           API_KEY: ${{ secrets.DGC_API_KEY }}
-        run: |
```

Comment:
Dart defines were hard to use to inject the client ID into the `oauth.html` plain web page. To use them we'd have to boot up Flutter which involves our own customised `flutter_bootstrap.js` and is rather slow in dev.

Instead I went with the "best worst" solution of putting everything back in the `.env` file and then fetching and reading it from JS in the oauth pages.

This shouldn't have any security implications as defines end up in the bundle just like packaging `.env` as an asset.